jvm-runtime/ftl-runtime/common/deployment/src/main/java/xyz/block/ftl/deployment/VerbProcessor.java

```diff
@@ -213,21 +157,36 @@ VerbClientBuildItem handleVerbClients(CombinedIndexBuildItem index, BuildProduce
                                 helper, publish.load(name), publish.load(module), publish.loadNull(),
                                 publish.loadClass(Void.class), publish.load(false), publish.load(false));
                         publish.returnVoid();
-                        clients.put(iface.name(), new VerbClientBuildItem.DiscoveredClients(name, module, cc.getClassName()));
-                    }
-                    found = true;
-                    break;
+                        break;
                 }
-            }
-            if (!found) {
-                throw new RuntimeException(
-                        "@VerbClientDefinition can only be applied to interfaces that directly extend a verb client type with concrete type parameters and "
-                                + iface.name() + " does not extend a verb client type");
+                clients.put(iface.name(), new VerbClientBuildItem.DiscoveredClients(name, module, cc.getClassName()));
             }
         }
         return new VerbClientBuildItem(clients);
     }
 
+    private MethodInfo getCallMethod(ClassInfo verbClient) {
+        for (var call : verbClient.methods()) {
```

Comment:
I think we need to throw an exception if the interface has additional non-static non-default methods, as they won't be implemented by the generated client.

Reply:
That would only happen if the user has mistakenly created their own @VerbClient interface right?